gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -0,0 +1,790 @@
+
+    return selects
+
+
+def custom_in_silico_select(ht: hl.Table) -> Dict[str, hl.expr.Expression]:
+    """
+    Get in silico predictors from VEP for release.
+
+    This function currently selects only SIFT and Polyphen from VEP.
+
+    :param ht: VEP Hail Table.
+    :return: Select expression dict.
+    """
+    vep_in_silico = get_sift_polyphen_from_vep(get_vep().ht())
+    selects = {
+        "sift_max": vep_in_silico[ht.key].sift_max,
+        "polyphen_max": vep_in_silico[ht.key].polyphen_max,
+    }
+    return selects
+
+
+def custom_region_flags_select(ht: hl.Table) -> Dict[str, hl.expr.Expression]:
+    """
+    Select region flags for release.
+
+    :param ht: Hail Table.
+    :return: Select expression dict.
+    """
+    selects = {
+        "region_flags": region_flag_expr(
+            ht,
+            non_par=True,
+            prob_regions={"lcr": lcr_intervals.ht(), "segdup": seg_dup_intervals.ht()},
+        )
+    }
+
+    return selects
+
+
+def custom_filters_select(ht: hl.Table) -> Dict[str, hl.expr.Expression]:
+    """
+    Select gnomAD filter HT fields for release dataset.
+
+    Extract "results" field and rename based on filtering method.
+
+    :param ht: Filters Hail Table.
+    :return: Select expression dict.
+    """
+    filter_name = hl.eval(ht.filtering_model.filter_name)
+    if filter_name == "RF":
+        name = "random_forest_results"
+    elif filter_name == "AS_VQSR":
+        name = "vqsr_results"
+    elif filter_name == "IF":
+        name = "isolation_forest_results"
+    else:
+        raise ValueError(f"Filtering method {filter_name} not recognized.")
+    selects = {name: ht.results.annotate(**ht.training_info)}
+
+    return selects
+
+
+def custom_info_select(ht: hl.Table) -> Dict[str, hl.expr.Expression]:
+    """
+    Select fields for info Hail Table annotation in release.
+
+    The info field requires fields from the freq HT and the filters HT so those are
+    pulled in here along with all info HT fields. It also adds the `allele_info` struct
+    to release HT.
+
+    :param ht: Info Hail Table.
+    :return: Select expression dict.
+    """
+    # Create a dict of the fields from the filters HT that we want to add to the info.
+    filters_ht = get_config().get("filters")["ht"]
+
+    score_name = hl.eval(filters_ht.filtering_model.score_name)
+    filters_ht = filters_ht.transmute(**filters_ht.truth_sets)
+    filters = filters_ht[ht.key]
+    filters_info_fields = [
+        "singleton",
+        "transmitted_singleton",
+        "omni",
+        "mills",
+        "monoallelic",
+        "only_het",
+    ]
+    filters_info_dict = {field: filters[field] for field in filters_info_fields}
+    filters_info_dict.update({**{f"{score_name}": filters[f"{score_name}"]}})
+
+    # Create a dict of the fields from the freq HT that we want to add to the info.
+    freq_ht = get_config().get("freq")["ht"]
+    freq_info_dict = {"inbreeding_coeff": freq_ht[ht.key]["InbreedingCoeff"]}
```

Comment:
Probably should have just changed this in the frequency code to be consistent. If we need to remake that table at all, we should change it